notebooks/405-paddleOCR-webcam/405-paddleOCR-webcam.ipynb

```diff
@@ -0,0 +1,466 @@
+    "\n",
+    "# get input and output nodes for text detection\n",
+    "det_input_layer = next(iter(det_compiled_model.inputs))\n",
+    "det_output_layer = next(iter(det_compiled_model.outputs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ec5c940-626c-4cf7-a90f-833200969846",
+   "metadata": {},
+   "source": [
+    "#### Load the Network for Text Recognition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89c0a07a-8186-47b5-ad95-f104a84d13d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rec_model_dir = \"./model/ch_ppocr_mobile_v2.0_rec_infer\"\n",
+    "rec_model_file_path = rec_model_dir + \"/inference.pdmodel\"\n",
+    "rec_params_file_path = rec_model_dir + \"/inference.pdiparams\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d196913-6542-4177-87ab-c5aa1994f8e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize the Paddle recognition inference on CPU\n",
+    "rec_ie = Core()\n",
+    "# read the model and corresponding weights from file\n",
+    "rec_net = rec_ie.read_model(model=rec_model_file_path, weights=rec_params_file_path)\n",
+    "\n",
+    "# assign dynamic shapes to every input layer on the last dimension\n",
+    "for input_layer in rec_net.inputs:\n",
+    "    input_shape = input_layer.partial_shape\n",
+    "    input_shape[3] = Dimension(-1)\n",
+    "    rec_net.reshape({input_layer: input_shape})\n",
+    "\n",
+    "rec_compiled_model = rec_ie.compile_model(model=rec_net, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes\n",
+    "rec_input_layer = next(iter(rec_compiled_model.inputs))\n",
+    "rec_output_layer = next(iter(rec_compiled_model.outputs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "573a1a11-faec-41af-bf43-08b90d28cec3",
+   "metadata": {},
+   "source": [
+    "### Preprocessing image functions for text detection and recognition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93bc8364-109b-4a32-b12b-bcb85f23b38c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def image_preprocess(input_image, size):\n",
+    "    img = cv2.resize(input_image, (size,size))\n",
+    "    img = np.transpose(img, [2,0,1]) / 255\n",
+    "    img = np.expand_dims(img, 0)\n",
+    "    # NormalizeImage: {mean: [0.485, 0.456, 0.406], std: [0.229, 0.224, 0.225], is_scale: True}\n",
+    "    img_mean = np.array([0.485, 0.456,0.406]).reshape((3,1,1))\n",
+    "    img_std = np.array([0.229, 0.224, 0.225]).reshape((3,1,1))\n",
+    "    img -= img_mean\n",
+    "    img /= img_std\n",
+    "    return img.astype(np.float32)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9329d709-14bc-45aa-a1d7-d0d6d608933b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Preprocess for Paddle Recognition\n",
+    "def resize_norm_img(img, max_wh_ratio):\n",
```

Comment:
Please use docstring format and type hints, for all classes, functions, methods. Here I leave you an example:
![image](https://user-images.githubusercontent.com/10940214/163566002-9e552d71-ba99-4d88-afb8-8c439b90aaac.png)